src/main/java/technology/semi/weaviate/client/v1/graphql/query/builder/AggregateBuilder.java

```diff
@@ -24,13 +25,15 @@ public class AggregateBuilder implements Query {
   NearTextArgument withNearTextFilter;
   NearObjectArgument withNearObjectFilter;
   NearVectorArgument withNearVectorFilter;
+  Integer objectLimit;
 
   private boolean includesFilterClause() {
     return withWhereArgument != null
       || StringUtils.isNotBlank(groupByClausePropertyName)
       || withNearTextFilter != null
       || withNearObjectFilter != null
-      || withNearVectorFilter != null;
+      || withNearVectorFilter != null
+      || objectLimit != null;
```

Comment:
since number of those != null checks gets higher, it might be worth to replace them with `ObjectUtils.anyNotNull()`

Reply:
Yes! good point!

Reply:
Nice, just pushed this